locale/manageiq.pot

```diff
@@ -83586,6 +84032,11 @@ msgstr ""
 msgid "target_class must be a class not an instance"
 msgstr ""
 
+#:
+#: ../app/javascript/components/workflow-credential-mapping-form/workflow-credential-mapping-form.schema.js
+msgid "test"
```

Comment:
@jeffibm Can you take a look at why this is showing up?  It shouldn't be here. See app/javascript/components/workflow-credential-mapping-form/workflow-credential-mapping-form.schema.js

Reply:
Oh I just noticed this is credential mapping - @DavidResende0 can you take a look?

Reply:
I found the problem, give me a second to fix

Reply:
@Fryguy, @jrafanie https://github.com/ManageIQ/manageiq-ui-classic/pull/8947

Reply:
Thanks!  Merged.

Reply:
ok, this is removed after this was merged.  Resolving.